CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ directly impact users rather than highlighting other key architectural updates.*
   lightwalletd servers in runtime.
 - The About screen now contains a link to the new Zashi Privacy Policy website
 
+### Changed
+- The Transaction History UI was incorporated into the Account screen
```

Comment:
FYI, I use present perfect everywhere instead of past. I suggest to change it into

`The Transaction History UI has been incorporated into the Account screen`